src/library/columncache.cpp

```diff
@@ -90,3 +93,100 @@ void ColumnCache::setColumns(const QStringList& columns) {
     m_columnSortByIndex.insert(m_columnIndexByEnum[COLUMN_PLAYLISTTRACKSTABLE_ARTIST], sortNoCase);
     m_columnSortByIndex.insert(m_columnIndexByEnum[COLUMN_PLAYLISTTRACKSTABLE_TITLE], sortNoCase);
 }
+
+void ColumnCache::slotSetKeySortOrder(double notation) {
```

Comment:
It looks like this is now only called if the notation changes. 
I think it should be called from the constructor as well.